delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storages/storage/model/StorageEvent.scala

```diff
@@ -1,4 +1,4 @@
-package ch.epfl.bluebrain.nexus.delta.plugins.storage.model
+package ch.epfl.bluebrain.nexus.delta.plugins.storages.storage.model
```

Comment:
I would recommend a different package structure, where the default package is (notice the singular):
```
ch.epfl.bluebrain.nexus.delta.plugins.storage
```
... and the model contains the file/storage sub-packages such that it follows the SDK structure:
```
ch.epfl.bluebrain.nexus.delta.plugins.storage.model.files
ch.epfl.bluebrain.nexus.delta.plugins.storage.model.storages
```